### ISSUE TITLE
Call to this._getFlowDirection(this.parentElement, true); overwrites flex-direction set in stylesheet

### ISSUE THREAD

**Issue by ppham27** [angular/flex-layout#364]
Commit #353 will set the flex-direction of the parent container to row when using fxFlex on a child element in this line. In our app, we set the flex-direction to column in a stylesheet, but the method only reads from the style attribute. Thus, the column flex-direction is overwritten.

This is where the child element sets the flex-direction of the parent element.

https://github.com/angular/flex-layout/blob/master/src/lib/flexbox/api/flex.ts#L162

My workaround has been to do

```js
@Component({                                                                                       
  moduleId: module.id,                                                                             
  selector: 'my-app',                                                                       
  host: {                                                                                          
    style: 'flex-direction: column',                                                               
  },                                                                                               
  styleUrls: ['./my_app.css'],                                                              
  templateUrl: './my_app.ng.html',                                                          
}) 
```

**Comment by rosslavery** [comment]
I believe this is a duplicate of #272.

**Comment by ThomasBurleson** [comment]
Fixed with SHA dde889d
